src/main/StarterTeam.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO Characters (name, element, birthplace) VALUES ('Traveler', 'ADAPTIVE', 'UNIVERSE');
```

Comment:
Neni tady jeste service, ale na 95% ti tohle spadne pri zavolani :) Can you tell me why?

Reply:
id?

Reply:
'ADAPTIVE'

Reply:
to jsem si taky všimla a uz pridala do enumu :0 
Ale trochu vic me trapi id. Snazila jsem se to udelat tak,  aby id generoval sam, ale nevim jestli timpadem mam davat slopec id a vymyslet id u ty inicializacni tabulky?